package.json

```diff
@@ -21,7 +21,7 @@
     "clean": "npm run clean:lib && npm run clean:bin",
     "lint": "coffeelint src spec",
     "coffee": "coffee --compile --output lib src",
-    "build": "npm run clean:lib && npm run coffee",
+    "build": "npm run clean:lib && npm run coffee && shx cp src/*.js lib",
```

Comment:
What is `shx`?

Reply:
It is a cross platform library with unix syntax. It already was used in other scripts above.

Reply:
It is something Amin added over at the atom-community/apm repo, I think.

It's useful because if you just throw shell commands in there, they are likely syntax errors in the Windows Command Prompt (cmd.exe) or Powershell. And vice-versa.

JSX allows certain essential Unix-style shell commands to be run, but it is implemented in pure JS I think. So it will run the same way on Windows, Linux, macOS.

In short: Lets you run Unix-style commands in your `package.json` scripts without them erroring out on non-Unix (i.e. without them erroring out on Windows).

Reply:
@DeeDeeG that actually can help me with some scripts too, so good to know this exist :)